src/Microsoft.Android.Sdk.ILLink/PreserveLists/Mono.Android.xml

```diff
@@ -7,17 +7,17 @@
 		<type fullname="Android.Runtime.InputStreamInvoker" preserve="methods" />
 		<type fullname="Android.Runtime.JNIEnv">
 			<method name="Exit" />
-			<method name="Initialize" />
 			<method name="PropagateUncaughtException" />
+		</type>
+		<type fullname="Android.Runtime.JNIEnvInit">
```

Comment:
I forget; does `JNIEnvInit` *need* to remain in `Mono.Android.dll`?  Or can it be moved to the assembly containing `AndroidRuntimeInternal`?

Reply:
It remains there for classic Xamarin, it is moved to the new assembly for .NET